Automattic/jetpack
# Extend ProductCard component and create recommendations UI

## Proposed changes:
- **Refactor ProductCard to ts, add 'recommendation' property**
- **Add price/discount calculation and checkout buttons to the product card variation**
- **Update useRecommendationsSection() to useEvaluationRecommendations()**

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
* Run JN ninja with Jetpack Beta
* Activate Jetpack plugin with `add/recommendations-section-component` branch
* Go to My Jetpack page, and go through onboarding process: Connect site, choose goals for your site
* As a result, you'll be presented "Our recommendations for you" section, you can confirm prices are correct, and links work well etc.
![CleanShot 2024-07-16 at 14 23 23@2x](https://github.com/user-attachments/assets/5bc86edb-9b47-4fc3-b2b3-4b8a7952abd3)
